### DISCUSSION TITLE
Live Stream Audio to C1310-e and 1410

### DISCUSSION THREAD

**Discussion by Watchnet1** [AxisCommunications/discussions#254] [General]
i am trying to find the correct method or application to use to live stream audio from  a desktop mic to an Axis IP  audio device, Speaker or encoder, any support or point in the right direction gracefully accepted

**Comment by vivekatoffice** [comment]
Related link:
[AXIS Audio Manager Pro](https://www.axis.com/products/axis-audio-manager-pro)
